module/Kanbanize/src/Kanbanize/Service/KanbanizeService.php

```diff
@@ -8,6 +8,8 @@
 
 	//public function moveTask(KanbanizeTask $kanbanizeTask, $status);
```

Comment:
Questi metodi sono chiamati all'interno del listener `Kanbanize\Service\SyncTaskListener` che adesso è disabilitato. Contengono le chiamate all'api di Kanbanize per spostare i task sulla board a seguito di un passaggio di stato in O.R.A. Contengono quindi una logica diversa da quella utilizzata per l'importazione: cosa è meglio fare in questi casi? Tenerli commentati oppure rimuoverli?